analysis/preregistration_template.Rmd

```diff
@@ -194,83 +219,90 @@ To be honest, I don't know how I might apply this to my work or any future works
 
 ### 4.3.2 Qualitative Model Checking	
 
 *This step is largely informal and case-specific, but requires ‚Äòface validation' with model users / clients / managers who aren't involved in the development of the model to assess whether the interactions and outcomes of the model are feasible an ddefensible  [@Grimm:2014es]. This process could be called a ‚Äúlaugh test" or a ‚Äúpub test, and in addition to checking the model's believability, it builds the client's confidence in the model [@Jakeman:2006ii].¬†Face validation could include structured walk-throughs, or presenting descriptions, visualisations or summaries of model results to experts for assessment. Briefly explain how you will qualitatively check the model, and whether + how you will include users and clients in the process.*
+HF- great section
 
-# 5.0 Model Validation and Evaluation	
+# 5.0 Model Validation and Evaluation
+HF- I think that this is covered in section 4 with descriptions of validation and checking. I'm not sure how this section is different.
 
 *Model Validation & Evaluation - This phase consists of a suite of analyses that collectively inform the decision and whether and when a model is suitable to meet its intended purpose (Augusiak, Van den Brink and Grimm 2014). Errors in design and implementation of the model and their implication on the model output are assessed. Ideally independent data is used against the model outputs to assess whether the model output behaviour exhibits the required accuracy for the model's intended purpose. The outcomes of these analyses build confidence in the model applications and increase understandingof model strengths and limitations. Model evaluation including model analysis and other testing should complement model checking. It should evaluate model checking, and consider over-fitting and extrapolation. The higher the proportion of calibrated, or uncertain parameters, the greater the risk that the model seems to work correctly [‚Ä¶] but for the wrong reasons" Evaluation thus complements model checking because we can rule out the chance that the model fits the calibration data well, but has not captured the relevant ecological mecahnisms of the system (Grimm). Evaluation of model outputs against external data in conjunction with the results from model checking provide information about the structural realism and therefore credibility of the model (Grimm et al. 2014).*
 
 ## 5.1 Model output corroboration	
-
+HF- This sounds like a second round of validation. I dont think I've ever seen this happen.
```

Comment:
I think it's a really interesting take on model development. If I understand it correctly it's basically a type of replication. However, unless this is way more common than I think, maybe it would be better to say something like:
"Are you planning on checking how your model predicts to a new dataset, if so please describe the process in detail"

Reply:
Also, sensitivity analyses can occur outside this context so maybe they should be shifted into a different section if you retain this - otherwise it implies that you wouldn't need to answer about sensitivity analysis if you are not going to test with a completely external dataset

Reply:
> I think it's a really interesting take on model development. If I understand it correctly it's basically a type of replication. However, unless this is way more common than I think, maybe it would be better to say something like:
> "Are you planning on checking how your model predicts to a new dataset, if so please describe the process in detail"

Ah ok, I understand where you are coming from now. That's not the intended meaning here at all. I think perhaps most ecologists will be inclined to use all their data to fit the model, and then just partition their full dataset to evaluate against those partitions using bootsrapping or cross-fold validation, and that's what I mean by 'independent' data set. Each time you test the data on the 'test' set, that counts as independent data. I think the term 'independent dataset' is obviously confusing, so it should probably be reworded for clarity.